cmd/bclient/bserver/chunkfile.go

```diff
@@ -0,0 +1,58 @@
+package bserver
+
+import (
+	"crypto/md5"
+	"fmt"
+	"io"
+	"os"
+	"path"
+)
+
+const BogusFileId string = "Uninitialized"
+
+func chunkAndUpload(fileroot string, srcFile string, srcFileMd5 []byte, item string, fileChunkSize int) (string, error) {
+
+	sourceFile, err := os.Open(path.Join(fileroot, srcFile))
+
+	if err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
+
+	defer sourceFile.Close()
+
+	chunk := make([]byte, fileChunkSize)
+
+	var fileId string = BogusFileId
+
+	// upload the chunk
+
+	for chunkNo := int64(1); chunkNo > 0; chunkNo++ {
```

Comment:
This for loop is funny since the condition is basically always going to hold (unless chunkNo rollsover, which I doubt).

While it is nice to see the loop increment in the for statement, I think it would be clearer if an explicit infinite loop construct is used instead, viz:

    var chunkNo int64
    for {
        chunkNo++
        // do something
    }

Looking at the loop, I don't think `chunkNo` is used at all. maybe it isn't needed?